x-pack/metricbeat/module/ibmmq/qmgr/manifest.yml

```
@@ -4,27 +4,7 @@ input:
   metricset: collector
   defaults:
     metrics_path: /metrics
-
-# The custom processor is responsible for filtering Prometheus metrics
-# not stricly related to the IBM MQ domain, e.g. system load, process,
-# metrics HTTP server.
-processors:
-  - script:
-      lang: javascript
-      source: >
-        function process(event) {
-          var metrics = event.Get("prometheus.metrics");
-          if (metrics == null) {
-            event.Cancel();
-            return;
-          }
-          Object.keys(metrics).forEach(function(key) {
-            if (!(key.match(/^ibmmq_.*$/))) {
-              event.Delete("prometheus.metrics." + key);
-            }
-          });
-          metrics = event.Get("prometheus.metrics");
-          if (Object.keys(metrics).length == 0) {
-            event.Cancel();
-          }
-        }
+    # Filtering out Prometheus metrics that are not strictly related to the
+    # IBM MQ domain, e.g. system load, process, metrics HTTP server.
+    metrics_filters:
+      include: ["ibmmq_.*", "up"]
```

```suggestion
      include: ["ibmmq_.*", "^up$"]
```

From https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-metricset-prometheus-collector.html#_filtering_metrics_2:
`To keep only specific metrics, anchor the start and the end of the regexp of each metric` ie `^node_network_up$`.